tests/_serialization/test_simple_properties.py

```diff
@@ -209,7 +209,7 @@ def test_union_runtime_errors():
         Union([Enumeration(BaseEnumeration)]).serialize(EnumerationExample.FOO)
     # The serialized type is correct (dict) but it is missing fields
     incomplete_dataset_dict = {'name': 'name'}
-    with pytest.raises(RuntimeError):
+    with pytest.raises(ValueError):
```

Comment:
This was no-covered in the code before, which I think is why it passed. I removed the no-cover and just updated the test.